src/rgw/rgw_lua_background.cc

```diff
@@ -135,7 +134,7 @@ void Background::run() {
   }
   try {
     open_standard_libs(L);
-    set_package_path(L, luarocks_path);
+  set_package_path(L, lua_manager->luarocks_path());
```

Comment:
nothing calls `set_package_path()` again if `RadosLuaManager::handle_reload_notify()` changes the path

Reply:
this is now moved inside the loop, we set the path before calling the script